answers/AbstractClassesAndMethods/0b8d656f-d71c-4787-9ba2-a16eaeeec79e/Person.php

```diff
@@ -0,0 +1,42 @@
+<?php
+abstract class Person
+{
+    protected $name;
+    private $age;
+	
+    public function __construct($name, $age)
+    {
+        $this->name = $name;
+        if ($this->checkAge($age)) {
```

Comment:
@TenshiLyle I used this private method `checkAge()` and `isValid()` to show learners the purpose of `private` keyword in the Visibility lesson. Since our lessons are connect with each other I also use the same methods that we have in the previous lessons that is why have these methods here...